### PR TITLE
ports over datasets

### LINKED ISSUE #2651: Port over Datasets
Needs https://github.com/space-wizards/space-station-14/pull/2566 for `DatasetPrototype`

Datasets in questions are
https://github.com/vgstation-coders/vgstation13/tree/Bleeding-Edge/config/names (except adjectives & verbs)
and 
https://github.com/vgstation-coders/vgstation13/tree/Bleeding-Edge/config/news

Every file should be made a corresponding dataset, like adjectives & verbs

### PR DESCRIPTION
**Pull request by PaulRitter**
fixes #2651

### CODE REVIEW

**Review by metalgearsloth — COMMENTED** [review]
Probably remove the ones we don't need yet (AI, borer, etc.) given they won't be used for years. Also add an issue after it gets merged to actually implement stuff like the name ones.

**Comment by PaulRitter** [comment]
I don't agree personally. I think its a good idea to not include job definitions of jobs that are not implemented etc. but with general word databases like these, i think its fine adding unused ones

**Comment by metalgearsloth** [comment]
@Zumorica can I get a vibe check on this.

File: Content.Client/EntryPoint.cs
Comment (Zumorica) [inline Content.Client/EntryPoint.cs:83]: Do we really want to ignore this client-side? I imagine it'd be best to simply not ship any server-only datasets with the client

**Review by Zumorica — APPROVED** [review]
I'd personally not ignore this client-side, other than that I see no issue with adding ones for stuff we don't have yet imo.

File: Content.Client/EntryPoint.cs
Comment (PaulRitter) [reply Content.Client/EntryPoint.cs:83]: this is due to the `DatasetPrototype` being in `Content.Server`. I could move it to shared if you want?

Comment (PJB3005) [reply Content.Client/EntryPoint.cs:83]: This prototype should 100% be on the client as well.

**Comment by PaulRitter** [comment]
moved dataset to shared as was asked, everything else was fine so i'll merge this in a week if no objections by then

**Pull request merged**
